util/MultiplayerCommon.h

```diff
@@ -51,6 +51,9 @@ FO_ENUM(
     ((GALAXY_SETUP_MEDIUM))
     ((GALAXY_SETUP_HIGH))
     ((GALAXY_SETUP_RANDOM))
+    ((GALAXY_SETUP_MICROSCOPIC))    // backwards-compatibility -> new enum values to the back
+    ((GALAXY_SETUP_TINY))          // backwards-compatibility -> new enum values to the back
+    ((GALAXY_SETUP_VERY_LOW))     // backwards-compatibility -> new enum values to the back
```

Comment:
in the next iteration there will be a dedicated enum for the monster frequencies, and maybe i'll even split it up into mobile monsters and stationary (guardian) monsters.